src/main/java/duke/AddCommand.java

```diff
@@ -0,0 +1,54 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+
```

Comment:
Perhaps you can have a Javadoc comment here briefly describing the class?

Reply:
Alright thanks